frontend/cs/scs/api.go

```diff
@@ -518,6 +518,51 @@ func (builder *builder) IsZero(i1 frontend.Variable) frontend.Variable {
 	return m
 }
 
+// IsNonZero returns 0 if a is zero, 1 otherwise
```

Comment:
didn't I kill that guy in the previous PR? :)

Reply:
`git merge` sucks